Assets/HoloToolkit-Examples/InteractiveElements/Scenes/InteractiveButtonComponents.unity

```diff
@@ -130,6 +147,11 @@ Prefab:
       propertyPath: m_RootOrder
       value: 4
       objectReference: {fileID: 0}
+    - target: {fileID: 114000010968933296, guid: 1208d39814b498d47a41952deff3abd0,
```

Comment:
Seems like this added an object reference, was this gameobject a prefab?  Do we need to be applying the changes to the prefab and not the scene?